docs/rename.md

```diff
@@ -0,0 +1,166 @@
+# Git Credential Manager Rename
+
+In November 2021, _"Git Credential Manager Core"_ was [renamed][rename-pr] to
+simply _"Git Credential Manager"_, dropping the "Core" moniker. We announced the
+new name in a [GitHub blog post][rename-blog], along with the new home for the
+project in it's own [organization][gcm-org].
+
+![Git Credential Manager Core renamed](img/gcmcore-rename.png)
+
+At the time, the actual exectuable name was not updated and continued to be
+`git-credential-manager-core`. As of [VERSION][rename-ver], the executable has
```

Comment:
The `VERSION` string is a placeholder for when we make the first release and know the version number of GCM.